packages/@ember/template/type-tests/index.test.ts

```diff
@@ -0,0 +1,13 @@
+import { SafeString } from '@ember/-internals/glimmer';
+import { htmlSafe, isHTMLSafe } from '@ember/template';
+import { expectTypeOf } from 'expect-type';
+
+let safe = htmlSafe('<div>someString</div>');
+expectTypeOf(safe).toEqualTypeOf<SafeString>();
+
+expectTypeOf(isHTMLSafe(safe)).toEqualTypeOf<boolean>();
```

Comment:
Question this raised for me: does `isHTMLSafe` narrow? If so, we should capture that in a test. If *not*, should we do a follow-on to *make* it narrow? I’m trying to think through whether/where that would be useful, but I can’t see that it would *hurt* anything, and I can imagine places where you might require a `SafeString` as an argument or some such. 🤔

Reply:
Looks like it is https://github.com/emberjs/ember.js/blob/8455848de50242657ae156848f05553a692afce1/packages/%40ember/-internals/glimmer/lib/utils/string.ts#L106